webapi/webapi-iap-xwalk-tests/index.html

```diff
@@ -0,0 +1,5 @@
+<!doctype html>
```

Comment:
I've tried not generated the index.html. However, this file can be generated only for the "root" apk, not for the sub apk.